Casks/w/workplace-chat.rb

```diff
@@ -8,10 +8,7 @@
   desc "Communications portal for your organisation"
   homepage "https://www.facebook.com/workplace/chat-app"
 
-  livecheck do
-    url :url
-    strategy :header_match
-  end
+  deprecate! date: "2025-02-15", because: :discontinued
```

Comment:
```suggestion
  # Workplace is being shut down - https://www.workplace.com/help/work/1167689491269151
  disable! date: "2026-05-31", because :discontinued
```

We can set the disable date now for when the service is no longer available, and it will also be immediately deprecated.

Reply:
It actually looks like the download is no longer available unless logged in. If there's no unauthenticated download available we can disable immediately.

Reply:
I installed an old version from web archives and found the Sparkle appcast, but it returns blank content, so I guess that we are no longer able to download it even if we authenticated. Let's disable immediately.

https://work.workplace.com/workplacechat/desktop/workatul/update.xml?target=workatul&platform=mac